easybuild/easyconfigs/a/Allinea/Allinea-6.1.1-Ubuntu-14.04-x86_64.eb

```diff
@@ -0,0 +1,26 @@
+name = 'Allinea'
+version = '6.1.1-Ubuntu-14.04-x86_64'
+
+homepage = 'http://www.allinea.com'
+description = """The Allinea environment is an essential toolkit for developers and computational scientists
+looking to get results faster."""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+source_urls = ['http://content.allinea.com/downloads/']
+sources = ['%(namelower)s-forge-%(version)s.tar']
+
+# Example of templates usage
+#templates = [
+#    'kebnekaise.qtf',
+#    'kebnekaise-gpu.qtf',
+#]
+
+# Example of sysconfig usage
+#sysconfig = 'system.config.hpc2n'
+
+# license file
+import os
+license_file = os.path.join(os.getenv('HOME'), "licenses", "allinea", "license.lic")
```

Comment:
@akesandgren please use

```python
license_file = HOME + 'licenses/allinea/license.lic'
```

using `import` in easyconfig files is something we should try and avoid..

do you mind also doing this in the existing easyconfig for Allinea?